packages/babel/package.json

```diff
@@ -44,7 +46,8 @@
   ],
   "peerDependencies": {
     "@babel/core": "^7.0.0",
-    "rollup": "^1.20.0||^2.0.0"
+    "rollup": "^1.20.0||^2.0.0",
+    "@types/babel__core": "^7.1.8"
```

Comment:
q: do we require this version specifically? wouldn't things work for `^7.0.0`?

Reply:
It need to be at least `7.1.9`, so the `hasFilesystemConfig` option exists.